posthog/models/team/team.py

```diff
@@ -280,3 +281,68 @@ def get_available_features_for_team(team_id: int):
     )
 
     return available_features
+
+
+def get_effective_membership_level(team_id: int, user_id: int) -> Optional["OrganizationMembership.Level"]:
```

Comment:
Did these methods move from somewhere else?

Did they change?

You'd hope we have strong tests around this 🤞

Reply:
Yeah, I have no idea. I'll have a look through the tests in the morning.

Reply:
And did they change? Yes I copied from the Team methods above and removed anything that relied on Django models.